### ISSUE TITLE
Error: Cannot find module 'node:buffer' on v3.6.9

### ISSUE THREAD

**Issue by pkdasari** [nodemailer/mailparser#362]
nodemailer version: `6.6.0`

Getting an error with the latest mailparser version v3.6.9

````javascript
const simpleParser = require("mailparser").simpleParser;

````

```
Error: Cannot find module 'node:buffer'
Require stack:
- /app/node_modules/mailparser/node_modules/libmime/lib/libmime.js
- /app/node_modules/mailparser/lib/mail-parser.js
- /app/node_modules/mailparser/index.js
```

**Comment by andris9** [comment]
What is your Node.js version?

**Comment by pkdasari** [comment]
Nodejs version: `14.15.5`

**Comment by andris9** [comment]
You need at least v16

**Comment by pkdasari** [comment]
@andris9 Thanks for quick response. Resolved the issue with mailparser version 3.6.7. 

No breaking changes are mentioned in the npm registry or release notes.

**Comment by andris9** [comment]
Node 14 is too old. I only mark a breaking change if it affects a supported version of Node, and not for old versions.
